samples/net/wpanusb/prj.conf

```diff
@@ -1,8 +1,9 @@
+CONFIG_USB_DEVICE_PID=0x101
+
 CONFIG_GPIO=y
 CONFIG_ARC_INIT=n
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
-CONFIG_SERIAL=y
```

Comment:
Why is CONFIG_SERIAL removed in a patch that only changed PID?

Reply:
That `CONFIG_SERIAL` gave warning that it was already set by other options, I included it in the same patch since the change is small, I can, of course, move this change to separate patch.

Reply:
It should rather have its own patch (or be a documented part of another); as it is in here it looks fishy.

Reply:
done

Reply:
Thanks, looks logical now.